docs/docs-content/clusters/edge/trusted-boot/deployment-day2/install.md

```diff
@@ -0,0 +1,179 @@
+   (VM), keys will always be auto-enrolled.
+
+5. If you did not set `AUTO_ENROLL_SECUREBOOT_KEYS` to `true` when you built the ISO, you will need to select the
+   **Enroll Secure Boot keys: Auto** option in the GRUB menu when the Edge host restarts. This will start the key
+   enrollment process.
+
+   :::warning
+
+   There is currently a known issue where if `AUTO_ENROLL_SECUREBOOT_KEYS` is set to `false`, it's possible for manual
+   key enrollment to take a few attempts to be successful.
+
+   :::
+
+### Install Palette Edge with Trusted Boot
+
+6. When the keys are finished enrolling, the installation will start automatically after a reboot. Installation
+   typically takes about 5 minutes, but can vary depending on your hardware.
+
+7. When installation finishes, you will observe the text "Installation has finished, rebooting in 5 seconds" on your
+   screen. Remove the USB disk from your Edge device.
+
+   :::info
+
+   If you have more than one boot devices, it is good security practice to go into the BIOS again and ensure that the
+   target installation disk is prioritized in the boot sequence.
+
+   :::
+
+8. After installation finishes, power up the device. Let your device automatically choose the option from the GRand
+   Unified Bootloader (GRUB) menu. It should boot directly to Edge device registration.
+
+## Validate
+
+Use the following steps to validate that Trusted Boot has been enabled on your Edge host. This requires you to have
+configured a user on your Edge host. Refer to [Installer Reference](../../edge-configuration/installer-reference.md) for
+more information.
+
+1. Press **Alt + right arrow key**, or **Ctrl + Alt + F1**. Replace **Alt** with **Options** on a Mac keyboard. This
+   will bring up a terminal and allow you to log in with the credentials you configured with the **user-data** file.
+
+   Alternatively, you can establish an SSH connection to your Edge device if you have network access to it.
+
+2. Issue the following command.
+
+   ```shell
+   ls -ltr /run/cos/
+   ```
+
+   You should receive an output that's similar to the following.
+
+   ```
+   total 12
+   -rwxr-xr-x  1 root root 1 Apr 30 15:59 uki_boot_mode
+   -rwxr-xr-x  1 root root  1 Apr 30 15:59 active_mode
+   -rw-r--r--  1 root root 534 Apr 30 15:59 cos-layout.env
+   ```
+
+   This confirms that your Edge device was booted with a Unified Kernel Image, a specific file format tailored to
+   achieve a tamper-proof system and encryption of persistent partitions.
+
+3. Issue the following command to verify disk encryption.
+
+   ```shell
+   lsblk
+   ```
+
+   You will observe output similar to the following and verify that the **/oem** portion and various directories
+   containing sensitive user data are encrypted. This means that if the boot process is ever altered by an unauthorized
+   party, they will not be able to gain access to the encrypted data.
```

Comment:
```suggestion
   An output similar to the following is displayed. Verify that the **/oem** portion and various directories
   containing sensitive user data are encrypted. This means that if the boot process is ever altered by an unauthorized
   party, they will not be able to gain access to the encrypted data.
```

@lennessyy looking the output, how can you determine it's encrypted? That was not intuitive to me.